MHVLib/Connection/MHVConnection.m

```diff
@@ -191,14 +207,68 @@ - (void)executeMethodRequest:(MHVMethodRequest *)request
         }
         else
         {
-            [self parseResponse:response request:request completion:request.completion];
+            [self parseResponse:response request:request isXML:YES completion:request.completion];
         }
-        
     }];
+}
+
+- (void)executeRestRequest:(MHVHttpServiceRequest *)request
+{
+    MHVRestRequest *restRequest = request.serviceOperation;
     
-    
+    if (restRequest.restRequestType == MHVRestRequestJSON)
+    {
+        [self executeJsonRestRequest:request];
+    }
```

Comment:
I'm not sure we should be doing this just yet - We should probably wait until after we determine the best approach for blobs before adding code to handle it.